modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/util/KBDropdownItemsProvider.java

```diff
@@ -973,6 +973,18 @@ private boolean _hasAddPermission() {
 		return false;
 	}
 
+	private boolean _hasChildArticles(KBArticle kbArticle) {
+		int childArticlesCount = KBArticleServiceUtil.getKBArticlesCount(
```

Comment:
my only question is about this change of name 

usually it's used _kbArticlesCount_ or something similar. but I do not know why you changed to _childArticlesCount_

It's a reason I miss?

same with `_hasKBChildArticles`

Reply:
To me it seems redundant to add the prefix `KB` when dealing with code in a module called _knowledge-base-web_ inside a class called _KBDropdownItemsProvider_ and having an input parameter called _kbArticle_ of type _KBArticle_. However, if guidelines say to use it I will do.

Reply:
It's better to change it.
And you do not have articles, you have kbArticles. 
An article can be of journal :S

Reply:
Renamed using standard guidelines, thanks for feedback